### PR TITLE
CICD Testing and Linting

### PR DESCRIPTION
**Pull request by 587900**

### CODE REVIEW

**Comment by 587851** [comment]
ok, but needs some work

**Comment by 587900** [comment]
noooo please

File: .github/workflows/cicd-test.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      with:
+        dotnet-version: 8.0.x
+    - name: Restore Dependencies
+      run: dotnet restore
+    - name: Build Microservices
+      run: dotnet build --no-restore
+    - name: Test Microservices
+      run: dotnet test --no-build --verbosity normal
+  test-client:
+    runs-on: ubuntu-latest
```
Comment (587851) [inline .github/workflows/cicd-test.yaml:23]: Ubuntu these nuts

**Review by 587851 — APPROVED** [review]
ok

**Pull request merged**